csrc/transform_replay.cpp

```diff
@@ -989,7 +989,7 @@ bool validateDomain(TensorView* tv, TensorDomain* new_td) {
 } // namespace
 
 void TransformPropagator::propagateC2P(TensorView* from, TensorView* to) {
-  int pos = replayed_pos_.at(from);
+  int pos = (int)replayed_pos_.at(from);
```

Comment:
Shouldn't we make `replayed_pos_` to hold `int` rather than `size_t`? @zasdfgbnm

Reply:
I agree. I think this makes more sense.